satpy/enhancements/contrast.py

```diff
@@ -38,6 +39,30 @@
 LOG = logging.getLogger(__name__)
 
 
+def noop_with_warning(img, msg=""):
+    """Perform no enhancement but emit a warning.
+
+    The warning message is specified by the ``msg`` keyword argument
+    and can be a format string. Keyword arguments will be passed
```

Comment:
There are [at least seven](https://stackoverflow.com/q/13451989/974555) ways to do string formatting in Python.  The documentation should specify which type is meant.  I think it's the type described as method 2b in the linked SO question.

Reply:
I think "format string" has a specific meaning and is similar to an f-string. That is, you call `.format` on it.